src/utils/i18n/enzymeHelper.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * Components using the react-intl module require access to the intl context.
+ * This is not available when mounting single components in Enzyme.
+ * These helper functions aim to address that and wrap a valid,
+ * English-locale intl context around them.
+ *
+ * copied from https://github.com/yahoo/react-intl/wiki/Testing-with-React-Intl#enzyme
+ */
+
+import React from 'react';
+import { IntlProvider, intlShape } from 'react-intl';
+import { mount, shallow } from 'enzyme';
+
+// Create the IntlProvider to retrieve context for wrapping around.
+const intlProvider = new IntlProvider({ locale: 'en', messages: {} }, {});
```

Comment:
I'm trying to understand what `messages` is here, and I can't really find it in the react-intl documentation. What does it do? It almost sounds like it is supposed to use the object in src/data/i18n/locales/{local}.json? Is that right? How does the tested component pull out the correct English string for testing if messages is empty?

Reply:
`messages` here should contain a mapping of {`id`: "`translated string value`"} pairs to use in translation. If a given id is not found in the messages dict (say, if messages is empty), react-intl will fallback to the defined `defaultMessage` value, which is always english in our case/when testing.

Reply:
Okay, great. Thanks! That's helpful.